integration_tests/pages/calendarPage.ts

```diff
@@ -22,7 +22,7 @@ export default class CalendarPage extends Page {
 
   bannerMFA = (): PageElement => cy.get('#banner-mfa')
 
-  notificationBannerLink = (): PageElement => cy.get('.govuk-notification-banner__link')
```

Comment:
after my changes I ended up with 3 occurrences of this on the page so difficult to tell which one was actually being targeted in the test